ocp_resources/resource.py

```diff
@@ -484,7 +484,7 @@ def wait(self, timeout=TIMEOUT, sleep=1):
         samples = TimeoutSampler(
             wait_timeout=timeout,
             sleep=sleep,
-            exceptions=(ProtocolError, NotFoundError),
+            exceptions_dict={**PROTOCOL_ERROR_EXCEPTION_DICT, NotFoundError: []},
```

Comment:
`NotFoundError: []` also appears more than once; please save and re-use

Reply:
done